Silicon/Qualcomm/sm8250/Include/Configuration/DeviceConfigurationMap.h

```diff
@@ -0,0 +1,46 @@
+#ifndef _DEVICE_CONFIGURATION_MAP_H_
+#define _DEVICE_CONFIGURATION_MAP_H_
+
+#define CONFIGURATION_NAME_MAX_LENGTH 64
+
+typedef struct {
+  CHAR8                        Name[CONFIGURATION_NAME_MAX_LENGTH];
+  UINT64                       Value;
+} CONFIGURATION_DESCRIPTOR_EX, *PCONFIGURATION_DESCRIPTOR_EX;
+
+static CONFIGURATION_DESCRIPTOR_EX gDeviceConfigurationDescriptorEx[] = {
+	{"AllowNonPersistentVarsInRetail", 0x1},
+	{"DloadCookieAddr", 0x01FD3000},
+	{"DloadCookieValue", 0x10},
+	{"EarlyInitCoreCnt", 1},
+	{"EnableDisplayImageFv", 0x1},
+	{"EnableDisplayThread", 0x1},
+	{"EnableLogFsSyncInRetail", 0x1},
+	{"EnableMultiThreading", 1},
+	{"EnableSDHCSwitch", 0x1},
+	{"EnableShell", 0x1},
+	{"EnableUfsIOC", 0},
```

Comment:
Is this configuration map from a 8250 uefiplat? This doesn't look right. If it's the case then there's going to be a need to reconfigure the IOMMU CB1 like on 855

Reply:
Yes, this is the one I extracted from uefiplat.cfg.

Reply:
ok then you may want to flip EnableUfsIOC to 1 and use the following code (with relevant addresses adjusted): https://github.com/edk2-porting/edk2-msm/blob/master/Silicon/Qualcomm/sm8150/Library/PlatformPrePiLib/PlatformUtils.c#L38-L53

Reply:
The addresses are in the header.